src/main/java/ua/yakaboo/pages/BasePage.java

```diff
@@ -0,0 +1,26 @@
+package ua.yakaboo.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.PageFactory;
+import utils.JSActions;
+import utils.Waits;
+
+public class BasePage {
+    protected WebDriver driver;
+    protected JSActions jsActions;
+    protected Waits waits;
+
+    public BasePage(WebDriver driver) {
+        this.driver = driver;
+        jsActions = new JSActions();
+        waits = new Waits(driver);
+        PageFactory.initElements(driver, this);
+    }
+
+    public static void clickButtonWithAction(WebDriver driver, WebElement element) {
```

Comment:
You already have WebDriver on this page, there is no need to pass through parameter method

Reply:
Updated